docker/docker-compose.setup.yml

```
@@ -10,4 +10,4 @@ services:
       - STATE=${STATE:-State}
       - CITY=${CITY:-City}
     volumes:
-      - ./secrets/certs:/certs:rw
+      - ../secrets/certs:/certs:rw
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)